index.js

```diff
@@ -0,0 +1,4 @@
+const { totalFuelForAllModulesOfMass } = require('./day-one-challenge/tyranny-of-the-rocket/getFuelRequirements');
+const moduleMassArray = require('./day-one-challenge/tyranny-of-the-rocket/fuelData');
+
+console.log(`day one: ${totalFuelForAllModulesOfMass(moduleMassArray)}`);
```

Comment:
I like it! `totalFuelForAllModulesOfMass` has no idea where its data is coming from. 💯